build.gradle

```diff
@@ -223,7 +223,7 @@ subprojects {
             // Keep the following references of tcnative version in sync whenever it's updated
             // SECURITY.md (multiple occurrences)
             // examples/build.gradle
-            netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:2.0.17.Final',
+            netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:2.0.20.Final',
```

Comment:
Sync these two: `SECURITY.md (multiple occurrences)` and `examples/build.gradle`

Reply:
Done.

Reply:
LGTM